clients/client-java/src/main/java/com/datastrato/gravitino/client/RelationalCatalog.java

```diff
@@ -277,6 +284,9 @@ static void checkTableNameIdentifer(NameIdentifier ident) {
     checkTableNamespace(ident.namespace());
   }
 
+  private Namespace getTableFullNamespace(Namespace tableNamespace) {
```

Comment:
There is a typo about `checkTableNameIdentifer` in line 280, can you fix it by the way?

Reply:
Thanks for pointing it out! I change it to "checkTableNameIdentifier" now

Reply:
Done